bitmap-ram-rod/lib/transformer.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const transform = module.exports = {};
+
+transform.modify = (parent, blueCallback, greenCallback, redCallback) => {
+
+  const helper = (bmp, position) => (callback) => (...ind) => {
```

Comment:
Like tis right?
`transform.modify = (parent, blueCallback, greenCallback, redCallback) => {

  const helper = (bmp, position, callback, ...ind) => {
    let hexColors = ind.map(hex => bmp.readUInt8(hex));

    return bmp.writeUInt8(callback(...hexColors), position);
  };
  let start = parent.colorTableStartPoint, end = parent.colorTableEndPoint;

  for (let i = 3 + start; i < end; i+=4) {
    helper(parent.buffer, i - 3, blueCallback, i-3, i-2, i-1);
    helper(parent.buffer, i - 2, greenCallback, i-2, i-3, i-1);
    helper(parent.buffer, i - 1, redCallback, i-1, i-2, i-3);
  }
};`